### DISCUSSION TITLE
[Kavita] Idea / Feature Submission Please Support folder depth deeper than 3!

### DISCUSSION THREAD

**Discussion by wangyeming** [Kareadita/Kavita#3344] [Ideas]
### Idea Description

I found the book wouldn't be scan by Kavita if the folder depth deeper than 3. And I had read the [guides](https://wiki.kavitareader.com/guides/scanner]) for Library Scanner. I want to Kavita can support more deeper folder scanner. For example the structures like this:

- Library Root
**** - Publisher A
******** - Author A
************ - Series Name A
************ - Series Name B
************ - Series Name C

But now, if the series were put on the folder which's hierarchy levels deeper than 3, they won't be scanned.
          

### Idea Category

API

### Duration of Using Kavita

10 days

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This is already possible. What's happening is you're hitting a bug in the v0.8.3 stable release that snuck in. Do a force library scan and when v0.8.4 releases, it will be fixed properly.

**Reply by wangyeming** [reply]
Thanks~
